src/api/status/public/js/utils/toast.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * UI theme for toast
+ * @typedef {("danger" | "success" | "info" | "warning" | "primary")} ToastVariant
```

Comment:
Really nice to see TS in JS, we should do more of this throughout Telescope.

Reply:
After working with the TypeScript codebase, I don't want any soul to go through the same thing again.

Reply:
It also adds a nice hint to the users.

![image](https://user-images.githubusercontent.com/52666982/145728075-576e03e8-dada-45e3-b0a7-779d60fce587.png)